docs/.vuepress/enhanceApp.js

```diff
@@ -0,0 +1,20 @@
+import * as Plugins from '../../src/plugins'
```

Comment:
Could you please run lint for this file to fix indentation and other styling issues.

Reply:
Fixed lint issues.